src/chrome/content/rules/Freifunk.xml

```diff
@@ -33,50 +28,71 @@
 
 	Fully covered subdomains:
 
+		- bug
 		- facebook	(→ www.facebook.com)
 		- forum
+		- blog.guetersloh
 		- hamburg
+		- pad
+		- paderborn
 		- twitter	(→ twitter.com)
 		- vimeo		(→ vimeo.com)
+		- wiki
 		- www
 		- youtube	(→ www.youtube.com)
 
 -->
 <ruleset name="Freifunk.net (partial)">
 
-	<target host="freifunk.net"/>
-	<target host="*.freifunk.net"/>
-		<!--
-			Redirect to http:
-						-->
-		<!--exclusion pattern="^http://freifunk\.net/$" /-->
-		<!--
-			Exceptions:
-					-->
-		<exclusion pattern="^http://freifunk\.net/+(?!favicon\.ico|wp-content/|wp-includes/)" />
+	<target host=                "freifunk.net"/>
+	<target host=            "bug.freifunk.net"/>
+	<target host=       "facebook.freifunk.net"/>
+	<target host=          "forum.freifunk.net"/>
+	<target host="blog.guetersloh.freifunk.net"/>
+	<target host=        "hamburg.freifunk.net"/>
+	<target host=            "pad.freifunk.net"/>
+	<target host=        "twitter.freifunk.net"/>
+	<target host=          "vimeo.freifunk.net"/>
+	<target host=           "wiki.freifunk.net"/>
+	<target host=            "www.freifunk.net"/>
+	<target host=        "youtube.freifunk.net"/>
+
+	<securecookie host="^(bug|forum|blog\.guetersloh|pad)\.freifunk\.net$" name=".+" />
+
+	<exclusion pattern="^http://(www\.)?freifunk\.net/+(?!favicon\.ico|wp-content/|wp-includes/)" />
```

Comment:
I can't see why that exclusion pattern exists. The site works perfectly on HTTPS. I think it was probably broken by the time the PR was created.